static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -426,20 +444,40 @@ function WidgetBuilder({
                 )}
               </div>
             </BuildStep>
-            <BuildStep
-              title={t('Group your results')}
-              description="Description of what this means"
-            >
-              WIP
-            </BuildStep>
+            {[DisplayType.TABLE, DisplayType.TOP_N].includes(state.displayType) && (
+              <BuildStep
+                title={t('Sort by')}
+                description="Description of what this means"
+              >
+                <Field
+                  inline={false}
+                  flexibleControlStateSize
+                  stacked
+                  error={state.errors?.orderby}
+                >
+                  <SelectControl
```

Comment:
Not an issue for this PR since more stuff will be added below this selector, might come up with future work but with more and more options in the sort by, the footer gets taller so they can be displayed
![Screen Shot 2022-02-16 at 9 23 37 AM](https://user-images.githubusercontent.com/63818634/154285378-65a0b013-2f2e-4ec4-911c-17f79bd8cb6e.png)

Reply:
yes we have noticed this too and will address these issues in a follow-up PR 😉